traffic.sh

```diff
@@ -0,0 +1,39 @@
+#!/usr/bin/env bash
+
+source "base_api.sh"
+source "utils.sh"
+
+function traffic_clones() {
+  local -r TOKEN="$1"
+  local -r OWNER="$2"
+  local -r REPO="$3"
+  body=$(base_api::get "$TOKEN" "repos/$OWNER/$REPO/traffic/clones")
+  echo "$body" | jq -r '.count, .uniques'
+  }
```

Comment:
Align the closing brackets, here and in other methods.

Reply:
Yes, of course, I’ll do it now

Reply:
@wardsenz Thanks!